pkg/core/common.go

```diff
@@ -54,3 +69,13 @@ func (config CommonAttackConfig) AttackKind() string {
 
 // CompleteDefaults no-op implementation
 func (config *CommonAttackConfig) CompleteDefaults() {}
+
+// Validate does a basic validation of common parameters
+func (config *CommonAttackConfig) Validate() error {
+	if len(config.Schedule) > 0 {
+		if dur, err := config.ScheduleDuration(); dur == nil || err != nil {
+			return errors.New("Provide a valid duration for the scheduled attack")
```

Comment:
```suggestion
			return errors.New("Provide a invalid duration for the scheduled attack")
```

Reply:
@WangXiangUSTC it should be "valid" only. The error states what the user should do to fix the error.

Reply:
OK, got it